src/en/components/date-input/base.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/base.njk'
+github: https://github.com/cds-snc/gcds-components/tree/main/packages/web/src/components/gcds-date-input
+figma: https://www.figma.com/design/mh2maMG2NBtk41k1O1UGHV/GC-Design-System?node-id=14220-878&node-type=canvas&t=TkjGZOzzUOp5XV3K-0
+permalink: false
+tags: ['dateinputEN', 'header']
+---
+
+# Date input <br>`<gcds-date-input>`
+
+_Also called: dates, dateinput, date, memorable date_
+
+A date input is a space to enter a known date.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Date input component preview" %}
+<gcds-date-input format="full" legend="Legend" name="example-default" hint="Day can be 1 or 2 digits. Year must be 4 digits.">
```

Comment:
Since we now have two date input's in this section are we able to use something other than "Legend" for both of them in the `legend` attributes?

Reply:
I'll wait for content and design to come up with something, but I flagged this in this comment: https://github.com/cds-snc/design-gc-conception/issues/1137#issuecomment-2369849311

If we follow the design it does show the two date inputs, but they both have the same "Legend" for the legend or label.

Reply:
We're removing it! Hopefully just having one that says "Legend" should be fine.